[Bug] build fail on build_ui

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

master

### What's Wrong?

```npm
npm ERR! Could not resolve dependency:
npm ERR! peer antd@"3.x" from @ant-design/compatible@1.1.2
npm ERR! node_modules/@ant-design/compatible
npm ERR!   @ant-design/compatible@"^1.0.2" from the root project
```

### What You Expected?

fix

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/ui/package.json
+++ b/ui/package.json
@@ -27,7 +27,6 @@
   "author": "lpx",
   "license": "ISC",
   "dependencies": {
-    "@ant-design/compatible": "^1.0.2",
     "@ant-design/icons": "^4.1.0",
     "@umijs/hooks": "^1.9.3",
     "antd": "^4.5.4",

